shlr/java/dsojson.c

```diff
@@ -580,6 +575,7 @@ R_API int dso_json_dict_insert_key_obj (DsoJsonObj *dict, DsoJsonObj *key, DsoJs
 		r_list_append (the_list, entry);
 		res = true;
 	} else {
+		dso_json_obj_del (value);
```

Comment:
@oddcoder why did you remove `dso_json_obj_del (val_obj)` and put it here but you didn't remove `dso_json_obj_del (key_obj)` ?